charts/fluid-datamigrate/juicefs/templates/service.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if gt (.Values.datamigrate.parallelism | int) 1 }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ printf "%s-svc" .Release.Name }}
```

Comment:
Can we also use name `{{ printf "%s-workers" .Release.Name }}`?

Reply:
fixed.